conformance/results/pyright/generics_syntax_infer_variance.toml

```diff
@@ -1,62 +1,66 @@
     Type parameter "K@ShouldBeInvariant3" is invariant, but "float" is not the same as "int" (reportAssignmentType)
-generics_syntax_infer_variance.py:127:43 - error: Type "ShouldBeInvariant3[str, int]" is not assignable to declared type "ShouldBeInvariant3[str, float]"
+generics_syntax_infer_variance.py:129:43 - error: Type "ShouldBeInvariant3[str, int]" is not assignable to declared type "ShouldBeInvariant3[str, float]"
   "ShouldBeInvariant3[str, int]" is not assignable to "ShouldBeInvariant3[str, float]"
     Type parameter "V@ShouldBeInvariant3" is invariant, but "int" is not the same as "float" (reportAssignmentType)
-generics_syntax_infer_variance.py:128:41 - error: Type "ShouldBeInvariant3[str, float]" is not assignable to declared type "ShouldBeInvariant3[str, int]"
+generics_syntax_infer_variance.py:130:41 - error: Type "ShouldBeInvariant3[str, float]" is not assignable to declared type "ShouldBeInvariant3[str, int]"
   "ShouldBeInvariant3[str, float]" is not assignable to "ShouldBeInvariant3[str, int]"
     Type parameter "V@ShouldBeInvariant3" is invariant, but "float" is not the same as "int" (reportAssignmentType)
-generics_syntax_infer_variance.py:136:38 - error: Type "ShouldBeInvariant4[int]" is not assignable to declared type "ShouldBeInvariant4[float]"
+generics_syntax_infer_variance.py:138:38 - error: Type "ShouldBeInvariant4[int]" is not assignable to declared type "ShouldBeInvariant4[float]"
   "ShouldBeInvariant4[int]" is not assignable to "ShouldBeInvariant4[float]"
     Type parameter "T@ShouldBeInvariant4" is invariant, but "int" is not the same as "float" (reportAssignmentType)
-generics_syntax_infer_variance.py:144:38 - error: Type "ShouldBeInvariant5[int]" is not assignable to declared type "ShouldBeInvariant5[float]"
+generics_syntax_infer_variance.py:146:38 - error: Type "ShouldBeInvariant5[int]" is not assignable to declared type "ShouldBeInvariant5[float]"
   "ShouldBeInvariant5[int]" is not assignable to "ShouldBeInvariant5[float]"
     Type parameter "T@ShouldBeInvariant5" is invariant, but "int" is not the same as "float" (reportAssignmentType)
-generics_syntax_infer_variance.py:155:45 - error: Type "ShouldBeContravariant1[int]" is not assignable to declared type "ShouldBeContravariant1[float]"
+generics_syntax_infer_variance.py:157:45 - error: Type "ShouldBeContravariant1[int]" is not assignable to declared type "ShouldBeContravariant1[float]"
   "ShouldBeContravariant1[int]" is not assignable to "ShouldBeContravariant1[float]"
     Type parameter "T@ShouldBeContravariant1" is contravariant, but "int" is not a supertype of "float"
       "float" is not assignable to "int" (reportAssignmentType)
 """
-conformance_automated = "Pass"
+conformance_automated = "Fail"
 errors_diff = """
+Line 18: Expected 1 errors
+Line 22: Expected 1 errors
+Line 17: Unexpected errors ['generics_syntax_infer_variance.py:17:42 - error: TypeVar cannot be both covariant and contravariant (reportGeneralTypeIssues)']
```

Comment:
These unexpected errors are caused by the reformatting in the test.

Reply:
Thank you for the feedback! I fixed the formatting issues.